mantidimaging/gui/windows/spectrum_viewer/test/model_test.py

```diff
@@ -28,6 +30,7 @@ class SpectrumViewerWindowPresenterTest(unittest.TestCase):
     def setUp(self) -> None:
         self.presenter = mock.create_autospec(SpectrumViewerWindowPresenter)
         self.model = SpectrumViewerWindowModel(self.presenter)
+        self.model.set_stack(generate_images())
```

Comment:
Can this go just in the tests that need it?

Reply:
My thinking was that its good have at least some tests see the `SpectrumViewerWindowModel` as it is when it is first created, otherwise we could end up with code that fails in real use because it assumed that there were always some ROIs in the model.

Reply:
I see. I'm happy to agree with that. My thinking was that if most tests are testing the roi dictionary when it is populated with "all" and "roi" because `set_stack` is called as part of setting up spectrum viewer and only a couple of tests test the roi dict when empty which is the rarer state that the roi dictionary will be in in real use to my understanding, to clear the dictionary only for those tests.